### ISSUE TITLE
Error raised when diff is empty

### ISSUE THREAD

**Issue by telyn** [ashfurrow/danger-rubocop#46]
Hey again Ash :)

We just implemented automated merge-backs where I work from our main branch to our release & develop branches - and immediately ran into a (very minor) issue: When the PR has commits to merge, but they result in an empty diff, danger-rubocop outputs the following error:

```
Danger::DSLError: 
[!] Invalid `Dangerfile` file: undefined method `patch' for nil:NilClass. Updating the Danger gem might fix the issue. Your Danger version: 7.0.0, latest Danger version: 8.2.1
 #  from Dangerfile:13
 #  -------------------------------------------
 #  
 >  rubocop.lint files: (git.modified_files + git.added_files),
 #               force_exclusion: true,
 #  -------------------------------------------
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:82:in `added_lines'
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:73:in `block in filter_out_offenses'
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:72:in `each'
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:72:in `filter_out_offenses'
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:66:in `rubocop'
  /opt/atlassian/pipelines/agent/build/vendor/bundle/ruby/2.6.0/bundler/gems/danger-rubocop-df966f15ffeb/lib/danger_plugin.rb:41:in `lint'
  Dangerfile:13:in `eval_file'
```

(Yes, sorry, still using our own custom version - hence the commit hash in the path - but I don't think anyone else has noticed, at least from a quick search through the issues)

We'll try to reproduce with master and get a PR in hopefully later this week.

**Comment by ashfurrow** [comment]
Huh! Interesting edge case. I agree this is a bug and should be fixed – a PR would be most welcome 🙇

**Comment by telyn** [comment]
My colleague @chocoberry4 says that the problem was fixed by switching to master - we'll keep an eye out to see if it reproduces but for now I think we can call this closed :heart:

**Comment by ashfurrow** [comment]
Okay, cool! I released version `0.9.3` just now with the latest `master` changes.
